### PR TITLE
4.0 ex expired non quantum

### PR DESCRIPTION
**Pull request by dmitryilyin**

### CODE REVIEW

File: deployment/puppet/cobbler/templates/scripts/pmanager.py [hunk 1]
```diff
@@ -304,8 +304,12 @@ def pvs(self):
                              begin_size, end_size, self.unit))
                 self.kick("partition {0} "
                           "--onpart=$(readlink -f /dev/{2}){3}{4}"
+<<<<<<< HEAD
```
Comment (bogdando) [inline deployment/puppet/cobbler/templates/scripts/pmanager.py]: yay! bad merging

**Comment by xarses** [comment]
looks like a rebase is needed

**Comment by aglarendil** [comment]
+2

**Comment by xarses** [comment]
Umm, why on earth did we merge this with the broken commit chain?

**Pull request merged**